packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -265,9 +265,40 @@ describe('PerformanceMonitoring', function() {
     expect(task.data.span.ended).toBeFalsy()
     var headerName = configService.get('distributedTracingHeaderName')
     var headerValue = getDtHeaderValue(task.data.span)
+    expect(req.setRequestHeader).toHaveBeenCalledTimes(1)
     expect(req.setRequestHeader).toHaveBeenCalledWith(headerName, headerValue)
   })
 
+  it('should add tracestate header when propagateTracestate is set', done => {
+    const patchFn = performanceMonitoring.getXHRSub()
+    configService.setConfig({ propagateTracestate: true })
+    const req = new window.XMLHttpRequest()
+    req.open('GET', '/', true)
+    spyOn(req, 'setRequestHeader').and.callThrough()
+    const task = {
+      source: XMLHTTPREQUEST,
+      data: {
+        target: req
+      }
+    }
+    req.addEventListener('readystatechange', function() {
+      if (req.readyState === req.DONE) {
+        patchFn('invoke', task)
+        expect(task.data.span.ended).toBeTruthy()
+        done()
+      }
+    })
+    patchFn('schedule', task)
+    req.send()
+    expect(task.data.span).toBeDefined()
+    const traceStateValue = getTSHeaderValue(task.data.span)
+    expect(req.setRequestHeader).toHaveBeenCalledTimes(2)
+    expect(req.setRequestHeader.calls.argsFor(1)).toEqual([
+      'tracestate',
+      traceStateValue
```

Comment:
It's better to hardcode the expected value here.

Reply:
Done